transforms/create-element-to-jsx.js

```diff
@@ -60,11 +60,20 @@ module.exports = function(file, api, options) {
     }
   };
 
+  const jsxIdentifierNameFor = node => {
```

Comment:
why do you need the recursion? Can't you just return "node" if its a MemberExpression and make the code accept either an AST node or a string?